.github/workflows/image.yaml

```diff
@@ -70,11 +70,15 @@ jobs:
       - name: Install cosign
         uses: sigstore/cosign-installer@main
         with:
-          cosign-release: 'v1.13.0'
+          cosign-release: 'v1.13.1'
+
+      - name: Install crane to get digest of image
+        uses: imjasonh/setup-crane@v0.1
```

Comment:
Can we pin this to a hash just for a little extra safety?

Reply:
Fixed as requested